azure-pipelines.yml

```diff
@@ -527,4 +527,4 @@ jobs:
     displayName: 'Checkin secrets'
     inputs:
       filePath: '$(Build.Repository.LocalPath)\build\checkin-secrets.ps1'
-      arguments: '-connectionString ''$(Storage-azurefunctionshostci0)'' -leaseBlob $(LeaseBlob) -leaseToken $(LeaseToken)'
\ No newline at end of file
+      arguments: '-connectionString ''$(Storage-azurefunctionshostci0)'' -leaseBlob $(LeaseBlob) -leaseToken $(LeaseToken)'
```

Comment:
I think GitHub did this automatically because I used the in-browser editor. Looks like it just added a newline.